cardano-testnet/cardano-testnet.cabal

```diff
@@ -188,14 +188,14 @@ test-suite cardano-testnet-test
                         Cardano.Testnet.Test.FoldBlocks
                         Cardano.Testnet.Test.Misc
 
-                        Cardano.Testnet.Test.LedgerEvents.Gov.DRepDeposits
-                        Cardano.Testnet.Test.LedgerEvents.Gov.InfoAction
-                        Cardano.Testnet.Test.LedgerEvents.Gov.ProposeNewConstitution
-                        Cardano.Testnet.Test.LedgerEvents.Gov.ProposeNewConstitutionSPO
-                        Cardano.Testnet.Test.LedgerEvents.Gov.TreasuryWithdrawal
-                        Cardano.Testnet.Test.LedgerEvents.Gov.DRepActivity
-                        Cardano.Testnet.Test.LedgerEvents.SanityCheck
-                        Cardano.Testnet.Test.LedgerEvents.TreasuryGrowth
+                        Cardano.Testnet.Test.EpochState.Gov.DRepDeposits
```

Comment:
Tbh I think mentioning EpochState package names is redundant. It's not like we're testing a different way of running node or something.

Reply:
Fair. So `Cardano.Testnet.Test.Gov.x`?